tools/make/lint.mk

```diff
@@ -68,6 +68,13 @@ lint.shellcheck: $(tools/shellcheck)
 	@$(LOG_TARGET)
 	$(tools/shellcheck) tools/hack/*.sh
 
+.PHONY: lint.fix-golint
+lint-deps: $(tools/gci)
+lint.fix-golint: ## Run all linter of code sources and fix the issues.
+	@$(LOG_TARGET)
+	$(MAKE) lint.golint GOLANGCI_LINT_FLAGS="--fix"
+	find . -name "*.go" | xargs $(tools/gci) write --skip-generated -s Standard -s Default -s "Prefix(github.com/envoyproxy/gateway)" 
```

Comment:
I thinks the `Prefix` should be consistent with L74 in `tools/linter/golangci-lint/.golangci.yml`